Source/Helika/Private/HelikaActor.cpp

```diff
@@ -11,206 +13,398 @@ AHelikaActor::AHelikaActor()
 void AHelikaActor::BeginPlay()
 {
     Super::BeginPlay();
-    _playerId = playerId;
-    Init(apiKey, gameId, helikaEnv, telemetry, printEventsToConsole);
+    Init(ApiKey, GameId, HelikaEnvironment, Telemetry, bPrintEventsToConsole);
```

Comment:
All variables are following PascalCase here but I wonder why it is not for `bPrintEventsToConsole`?